### PR TITLE
[Filter] reduce complexity

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, add internal function for filter properties.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by gichan-jang — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -297,10 +299,59 @@ gst_tensor_filter_get_dimensions_string (GstTensorFilterProperties * prop,
...
 }
 
+/**
+ * @brief Get the type string of tensors.
+ * @param[in] prop GstTensorFilterProperties object
+ * @param is_input TRUE if target is input tensors
+ * @return type string of tensors
+ */
+static gchar *
+gst_tensor_filter_get_types_string (GstTensorFilterProperties * prop,
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:316]: add const to prop?

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 2]
```diff
@@ -297,10 +299,59 @@ gst_tensor_filter_get_dimensions_string (GstTensorFilterProperties * prop,
...
+
+/**
+ * @brief Get the name string of tensors.
+ * @param[in] prop GstTensorFilterProperties object
+ * @param is_input TRUE if target is input tensors
+ * @return name string of tensors
+ */
+static gchar *
+gst_tensor_filter_get_names_string (GstTensorFilterProperties * prop,
+    const gboolean is_input)
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:340]: Add const to prop?

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 3]
```diff
@@ -330,26 +381,36 @@ gst_tensor_get_layout_string (tensor_layout layout)
...
  */
 static gchar *
-gst_tensors_get_layout_string (const GstTensorsInfo * info,
-    tensors_layout layout)
+gst_tensor_filter_get_layouts_string (GstTensorFilterProperties * prop,
+    const gboolean is_input)
 {
   gchar *layout_str = NULL;
+  GstTensorsInfo *info;
+  tensors_layout *layout;
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:389]: Add const to prop, info, layout??

**Review by myungjoo — COMMENTED** [review]
LGTM in overall. Please consider constifying struct parameters

**Comment by jaeyun-jung** [comment]
2020-12-10 PR updated to reduce duplicated code when setting properties

Please see the changes again.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**